.github/workflows/docker-apply-production.yml

```
@@ -23,6 +23,8 @@ jobs:
         include:
           - export: platform/gc_notify
             lambda: platform-gc-notify-export
+          - export: platform/gc_design_system/airtable
+            lambda: platform-gc-design-system-export
```

Could you tweak this name to add airtable since we'll have another one for the npm stats:
```suggestion
            lambda: platform-gc-design-system-airtable-export
```